lib/providers/add_excercise_provider.dart

```diff
@@ -2,9 +2,31 @@ import 'package:flutter/foundation.dart';
 
 import 'dart:io';
 
+import 'package:wger/models/exercises/muscle.dart';
+
 class AddExcerciseProvider with ChangeNotifier {
   List<File> get excerciseImages => [..._excerciseImages];
   final List<File> _excerciseImages = [];
+  String? _name;
+  String? _alternativeName;
+  String? _targetArea;
+  List<String?>? _primaryMuscles = [];
```

Comment:
wouldn't it be better to store this a a list of Muscles? Same goes to the other stuff we get from the form, it will be easier for us later when we need to do work with those objects later

Reply:
Currently DropDowns and MultiSelect widgets are designed to handle only Strings and not Object, that is the reason I kept it as string.

Reply:
Ah, ok. Can we store the object IDs and do a lookup when saving?

Reply:
that's possible , but then in UI we will only see ID and not name.
Instead we need to re-design the widgets.

Reply:
Showing only the IDs is not optimal 😄, I guess we will have to think of something

Reply:
Cool then we need to re-design the widgets,
I'll try to do them.